extensions/aws/src/test/java/io/opentelemetry/extension/aws/AwsXrayPropagatorTest.java

```diff
@@ -168,11 +189,12 @@ void extract_AdditionalFields() {
         TRACE_HEADER_KEY,
         "Root=1-8a3c60f7-d188f8fa79d48a391a778fa6;Parent=53995c3f42cd8ad8;Sampled=1;Foo=Bar");
 
-    // TODO: assert additional fields when the propagator supports it
-    assertThat(getSpanContext(xrayPropagator.extract(Context.current(), carrier, getter)))
+    Context context = xrayPropagator.extract(Context.current(), carrier, getter);
+    assertThat(getSpanContext(context))
         .isEqualTo(
             SpanContext.createFromRemoteParent(
                 TRACE_ID, SPAN_ID, TraceFlags.getSampled(), TraceState.getDefault()));
+    assertThat(Baggage.fromContext(context).getEntryValue("Foo")).isEqualTo("Bar");
```

Comment:
any worries with people putting your keys into the Baggage by accident? So, a baggage entry called `Root` or `Sampled` or `Parent` ?

Reply:
Ah yeah we'd want to ignore it (I don't think the x-ray sdk is so sophisticated though...). I think that's what happens will add unit test.